sklearn/model_selection/_validation.py

```diff
@@ -210,13 +211,22 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     parameters : dict or None, optional
         The parameters that have been evaluated.
     """
+    current_fit_msg = ""
+    if verbose > 2:
+        if split_progress:
+            current_fit_msg = " split %d of %d;" % split_progress
+        if param_progress:
+            current_fit_msg += " candidate %d of %d" % param_progress
+
     if verbose > 1:
         if parameters is None:
-            msg = ''
+            params_msg = 'params - {}'
```

Comment:
This isn't correct. The parameters is None handles something like `cross_val_score` or `learning_curve` where parameters are invariant/irrelevant.